src/core/api/rtd_csv_transaction/create-sas-token-policy.xml

```diff
@@ -63,18 +63,27 @@
 
             See: https://stackoverflow.com/questions/38474362/get-a-file-sha256-hash-code-and-checksum
         -->
-        <set-variable name="senderCode" value="@(((IResponse)context.Variables["senderCodeResponse"]).Body.As<string>())"/>
 
-        <set-variable name="senderCodeHash" value="@{
+        <choose>
+            <when condition="@(((IResponse)context.Variables["senderCodeResponse"]).StatusCode == 200)" >
+                <set-variable name="senderCode" value="@(((IResponse)context.Variables["senderCodeResponse"]).Body.As<string>())"/>
```

Comment:
@TommasoLencioni If the association `SenderCode <--> Api Key` is not unique, as we are planning, how can we decide which sender code is the _master_ one?

Reply:
That's a good point, we should discuss about it.